src/libPMacc/include/math/vector/math_functor/sqrtf.hpp

```diff
@@ -37,9 +37,9 @@ struct Sqrtf
 {
     typedef float result_type;
 
-    HDINLINE float operator()(const float& value) const
```

Comment:
We should bring this to a more generic version like https://github.com/ComputationalRadiationPhysics/picongpu/pull/847/files#r29922579